### PR TITLE
Readme link

### PR DESCRIPTION
**Pull request by jpjoyal**

### CODE REVIEW

**Comment by jpjoyal** [comment]
@dannytranlx

**Comment by dannytranlx** [comment]
gosh why you merge so fast :|

File: README.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
 # SupportKit
 *Beautifully simple in-app messaging*
 
-SupportKit is a free iOS SDK that connects you to your users so you can communicate with them and build a killer app. If you want to learn more about SupportKit, then visit [our website](https://supportkit.io) or [sign up here](https://app.supportkit.io/signup). This project provides in-depth documentation on SupportKit’s features and is meant to be a guide for developers integrating the SDK and app makers seeking to engage with their users.
+This projects hosts the documentation available at [http://docs.supportkit.io/](http://docs.supportkit.io/).
```
Comment (dannytranlx) [inline README.md]: project*

**Comment by jpjoyal** [comment]
My bad!

**Pull request merged**